src/chigame/games/models.py

```diff
@@ -223,24 +224,57 @@ def __str__(self):  # may be changed later
         )
 
 
-class Notification(models.Model):
+class Announcement(models.Model):
     """
-    A notification, which can be sent to multiple users.
+    An announcement, which can be sent to multiple users.
     """
 
-    recipients = models.ManyToManyField(User, related_name="notifications")
+    REMINDER = 2
+    UPCOMING_MATCH = 3
+    MATCH_PROPOSAL = 4
+
+    ANNOUNCEMENT_TYPES = (
+        (REMINDER, "REMINDER"),
+        (UPCOMING_MATCH, "UPCOMING_MATCH"),
+        (MATCH_PROPOSAL, "MATCH_PROPOSAL"),
+    )
```

Comment:
Do we want a notification type for letting users know it's time to join a tournament? Or are we putting that into the upcoming match notification?

Reply:
We can put that in the upcoming_match notification type. We only have three types here as we are actually basing our announcement model on the notification model from users/model.py, which only provides these options and one inapplicable friend request option.